frontend/src/app/main/data/comments.cljs

```diff
@@ -75,9 +75,9 @@
 
     (ptk/reify ::create-comment-thread
       ptk/WatchEvent
-      (watch [_ _ _]
-        (->> (rp/mutation :create-comment-thread params)
-             (rx/mapcat #(rp/query :comment-thread {:file-id (:file-id %) :id (:id %)}))
+      (watch [_ state _]
+        (->> (rp/mutation :create-comment-thread (assoc params :share-id (get-in state [:viewer-local :share-id])))
```

Comment:
I think, this assoc params can be extracted to 

```clojure
(let [share-id (-> state :viewer-local :share-id) 
      params (assoc params :share-id share-id)]
...)
```

And avoid large lines, creating unnecessary vectors and avoid repeating operations